### ISSUE TITLE
Ripple effect goes outside Button

### ISSUE THREAD

**Issue by praveenbroski** [traex/RippleEffect#89]
I used your library and it's very good. 
Thank you for your efforts. But I'm facing an issue that the ripple effects goes outside the button. I mean it forms a boundary out side the button like button is with in a rectangular box. And it doesn't look nice to me. Do you have any workaround to solve that. I'm expecting your support soon. Thanks in advance.

**Comment by abhinavnair** [comment]
Yeah same here. The ripple goes outside the button as if there is the button is enclosed in another invisible rectangular container. Clicking just above the button also causes the same effect.

**Comment by abhinavnair** [comment]
Found the problem. I had given a margin to my button. Once the margin is removed, it works perfectly fine. If you want to give a margin you can give to the RippleView directly.
